Makefile

```diff
@@ -11,19 +10,20 @@ TARGET_DIR := ./target
 BIN_NAME := attestation-agent
 
 DEBUG ?=
-MUSL ?=
+LIBC ?=
```

Comment:
various files in .github/workflows use `MUSL=1` in some places. These should be updated to the new syntax.

Reply:
Done.